docker/README.md

```diff
@@ -0,0 +1,95 @@
+# Docker container description for Tendermint applications
+
+* Overview (#Overview)
+* Tendermint (#Tendermint)
+* Basecoin (#Basecoin)
+* Ethermint (#Ethermint)
+
+## Overview
```

Comment:
We did this for the hackethon. You can have a look. We will finalize it tomorow:
https://github.com/multiverseHQ/deployment/tree/master/docker_compose